applications/project_aurras_mvp_phase_2.md

```diff
@@ -0,0 +1,239 @@
+# W3F Grant Proposal
+
+- **Project Name:** Project Aurras - MVP - Phase 2
+- **Team Name:** HugoByte AI Labs Private Limited
+- **Payment Address:** 1G5Fo5KfLSopTFKPRDUZgeCSU5PNY2kkD5 (BTC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+This is an application for a follow-up grant: https://github.com/w3f/Grants-Program/pull/163
+
+### Overview
+
+Aurras is a middleware that acts as an event processor and a low code workflow orchestration platform. Aurras is being pitched as a next-generation system for enabling decentralized push notification. This middleware solution listens to events from blockchain applications and propagates them to a registered pool of MQTT brokers. The broader architecture consist of parachain from which the middleware listens for the events.
+
+Aurras as a part of Substrate / Kusama / Polkadot / Web 3 Ecosystem
+* A core part of Decentralized Push Notification Architecture
+* Can be used for business-critical and production-grade IoT devices to listen to events from blockchain
+* Can be used with microservices across industry verticals
+* Can be expanded to get integrated with data analytics, knowledge visualization, and machine and deep learning (ML/DL) algorithms
+
+### Project Details
+
+#### Architecture
+![](https://storage.googleapis.com/hugobyte-2.appspot.com/architecture.jpeg)
+
+#### Flow Diagram
+![](https://storage.googleapis.com/hugobyte-2.appspot.com/aurras-flow-daigram.jpeg)
+
+#### Technologies
+
+1. [Apache OpenWhisk](https://openwhisk.apache.org) is an open source, distributed Serverless platform that executes functions (fx) in response to events at any scale. OpenWhisk manages the infrastructure, servers and scaling using Docker containers. The OpenWhisk platform supports a programming model in which developers write functional logic (called Actions), in any supported programming language, that can be dynamically scheduled and run in response to associated events (via Triggers) from external sources (Feeds) or from HTTP requests
+2. [Apache Kafka](https://kafka.apache.org) is an open-source distributed event streaming platform. Kafka works well as a replacement for a more traditional message broker. Message brokers are used for a variety of reasons (to decouple processing from data producers, to buffer unprocessed messages, etc.). In comparison to most messaging systems, Kafka has better throughput, built-in partitioning, replication, and fault-tolerance which makes it a good solution for large scale message processing applications.
+3. [Tackle box](https://robcxyz.github.io/tackle-box/) is a declarative DSL for building modular utilities and code generators. Tool is plugins based and can easily be extended by writing additional hooks or importing external providers creating a web of interoperable CLIs.
```

Comment:
By "Tool is plugins based" do you mean "the tool (tackle box) is plugin-based"?

Reply:
Yes the tool is tackle box we will update the application to remove this confusion, Thank you for pointing it out.